shellcheck.properties

```diff
@@ -1,14 +1,20 @@
 category=Languages
 description=Enables scanning of shell scripts
 homepageUrl=https://github.com/sbaudoin/sonar-shellcheck
-archivedVersions=2.0,2.1.0
-publicVersions=2.2.0
+archivedVersions=2.0,2.1.0,2.2.0
+publicVersions=2.2.1
 
 defaults.mavenGroupId=com.github.sbaudoin
 defaults.mavenArtifactId=sonar-shellcheck-plugin
 
+2.2.1.description=Fixes an issue that could prevent the plugin from being installed in some SonarQube environments running MySQL
+2.2.1.sqVersions=[6.7,LATEST]
```

Comment:
Please end this range at `7.9.*`, per 
https://community.sonarsource.com/t/new-release-shellcheck-plugin-2-2-1/15686/3